src/chrome/content/rules/iTerm2.xml

```diff
@@ -0,0 +1,24 @@
+<!--
+	For iterm2.com
+
+		Works:
+
+			- www
+			- $
+
+		Doesn't Work:
+			nil
+
+		www redirects to $
+		doing rewrite here for performance.
+-->
+<ruleset name="iTerm2">
+  <target host="iterm2.com" />
+  <target host="*.iterm2.com" />
+
+  <rule from="^http://(?:www\.)?iterm2\.com/"
+          to="https://iterm2.com/" />
+
+  <test url="http://iterm2.com/" />
+  <test url="http://www.iterm2.com/" />
```

Comment:
What I've been doing with rulesets like these, where there's a wildcard but the actual rules only rewrite the base name + www, is to replace the wildcard with "www." That takes advantage of the implicit test URL to save some typing, and make it a little clearer to immediately see which domains are rewritten.

In some cases, like this one, narrowing down the targets like that lets you dramatically simplify the rule to this:

```
<rule from="^http:"
  to="https:"
```

You lose the redirect-skipping performance gain you mentioned in the comment, but since HTTPS Everywhere is not intended to try and improve end-user performance, I think the gain in ruleset simplicity is worth it.